packages/http-client-java/generator/http-client-generator-clientcore-test/src/test/java/payload/xml/XmlTests.java

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package payload.xml;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public final class XmlTests {
+
+    @Test
+    public void testSimpleModel() {
+        SimpleModelValueClient client = new XmlClientBuilder().buildSimpleModelValueClient();
+
+        client.put(new SimpleModel("foo", 123));
```

Comment:
Do we need to verify the serialized result?

Reply:
It would be in the test of GET. This need to wait for new release of annotation-processor due to bug there.